compiler/src/iree/compiler/Codegen/VMVX/LowerLinalgMicrokernels.cpp

```diff
@@ -912,6 +912,48 @@ bool isMmt4d(ArrayAttr indexingMaps) {
   return indexingMaps == maps;
 }
 
+int getNumberOfUses(Value v) {
```

Comment:
Nit: Mark this `static`.

Reply:
This is already part of an anonymous namespace and there are already functions in there not marked `static`, such as `permuteStrides`.